server/src/main/java/org/opensearch/index/remote/RemoteRefreshSegmentTracker.java

```diff
@@ -237,6 +273,16 @@ public void updateRemoteRefreshTimeMs(long remoteRefreshTimeMs) {
         computeTimeMsLag();
     }
 
+    public void updateRemoteRefreshClockTimeMs(long remoteRefreshClockTimeMs) {
+        assert remoteRefreshClockTimeMs >= this.remoteRefreshClockTimeMs : "newRemoteRefreshClockTimeMs="
```

Comment:
Can we remove this assert? This is probably going to be flaky. The time returned by `System.currentTimeMillis()` is not monotonic. Reference - https://stackoverflow.com/questions/2978598/will-system-currenttimemillis-always-return-a-value-previous-calls#:~:text=Technically%2C%20it's%20monotonically%20increasing.

Reply:
yeah, will remove